jenkins/ci.suse.de/cloud-ardana8.yaml

```diff
@@ -1,34 +1,34 @@
 - project:
-    name: cloud-ardana8-gating-trigger
-    ardana_url_trigger_job: '{name}'
+    name: cloud-8-gating-trigger
+    url_trigger_job: '{name}'
```

Comment:
This parameter has to be the same as the one you use in `cloud-url-trigger-template.yaml` (in this case `cloud_url_trigger_job`). Running a JJB test should reveal this as an error.